app.py

```diff
@@ -1,12 +1,115 @@
 import streamlit as st
+import streamlit.components.v1 as components
 from pathlib import Path
 import json
-# For some reason the windows version only works if this is imported here
-import pyopenms
+import os
+import time
+import threading
+import signal
+import tornado.web
```

Comment:
need to import? https://github.com/OpenMS/streamlit-template/blob/main/requirements.txt